docs/backend-api/README.md

```diff
@@ -13,82 +13,122 @@ Inside a _tenant namespace_ the client creates `pipelinerun` resources for each
 
 ## Tenant Resource
 
-### Create
+### Spec
+
+#### Examples
 
 A simple `Tenant` resource example can be found in [docs/examples/tenant.yaml](../examples/tenant.yaml).
 
-| Parameter | Description |
+
+#### Fields
+
+| Field | Description |
 | --------- | ----------- |
-|`metadata.name`     | the resource name has to be the unique tenant ID |
+| `apiVersion` | `steward.sap.com/v1alpha1` |
+| `kind` | `Tenant` |
+| `metadata.name` | The resource name has to be the unique tenant ID. |
 
-```bash
-$ kubectl apply -f tenant.yaml
-```
 
-### Read
+### Status
 
-The tenant resource in Kubernetes is enriched with a `status` while it is processed and when tenant preparation finished.
+The `status` section of the `Tenant` resources lets clients know about the tenant namespace assigned exclusively to a tenant.
+
+After a client created a new `Tenant` resource, the Steward controller tries to achieve the hereby requested state:
+
+- A tenant namespace exists that is exclusively assigned to this tenant.
+
+- Service account `<tenant_namespace>::default` (where `<tenant_namespace>` is the name of the namespace assigned exlusively to the tenant) has the permissions needed to manage further resources in the tenant namespace.
+
+- Service account `<client_namespace>::default` (where `<client_namespace>` is the namespace where the `Tenant` resource belongs to) has the permissions needed to manage further resources in the tenant namespace.
+
+The Steward controller periodically checks the actual state of the resource and tries to change it to the desired state:
+
+- The role binding in the tenant namespace gets updated/recreated if needed, for instance if the client namespace's annotation `steward.sap.com/tenant-role` (defining the RBAC role to be assigned to the above-mentioned service accounts) has changed.
+
+- If `status.tenantNamespaceName` refers to a namespace that does not exist anymore, the ready condition is set to `False` indicating that the tenant is no longer ready to be used. As this never happens under normal circumstances and probably means that data has been lost, the tenant namespace will not be recreated automatically. Operators should monitor tenants, and must  analyze and fix the underlying issue if such situations occur.
+
+
+### Examples
 
-```bash
-$ kubectl -n <steward-client1> get tenant <tenantId> -oyaml
-```
-_(shortened example yaml)_
 ```yaml
+apiVersion: steward.sap.com/v1apha1
+kind: Tenant
+metadata:
+  name: tenant1
+  namespace: steward-c-client1
 status:
-  message: Tenant namespace successfully prepared
-  progress: Finished
-  result: success
-  tenantNamespaceName: stu-tn-cl1-test-tenant-09a530
+  conditions:
+  - type: Ready
```

Comment:
maybe an error example with a `message`?

Reply:
I've added error examples in 9e59518d068877c457b5a7b64dc9d9ea0a1db5b9.